_includes/assets/css/inline.css

```diff
@@ -385,54 +333,226 @@ form[name='contact'] textarea {
 
 a[href$='#subscribe'] {
   color: #011;
-  background: linear-gradient(-45deg,#ff0,#ff6f00,#ffa000,#ff0,#ff8200);
+  background: linear-gradient(-45deg, #ff0, #ff6f00, #ffa000, #ff0, #ff8200);
   background-size: 400% 400%;
   animation: gradient 30s ease infinite;
   padding: 0.33rem;
   display: inline-block;
   text-align: center;
   margin: 0.25rem;
   border-radius: 0.5rem;
-  transition: transform .2s ease-in-out;
+  transition: transform 0.2s ease-in-out;
 }
 
 a[href$='#subscribe']:hover {
   color: #229;
   transform: scale(1.2);
 }
 
+#featureTabList li:hover,
+#tabList li:hover {
+  background-color: #f7d38a;
+}
+
+#featureTabList li,
+#tabList li {
+  text-decoration: underline;
+}
+
+li.selected {
+  background-color: #edbf63 !important;
+}
+
+#featureTabContent article:not(.active),
+#tabContent article:not(.active) {
+  display: none;
+}
+
+/* blog post starts */
+article.post {
+  max-width: 830px;
+}
+
+article.post h1 {
+  font-family: 'Geist';
+  font-style: normal;
+  font-weight: 650;
+  font-size: 52px;
+  line-height: 57px;
+  letter-spacing: -0.04em;
+  margin: 10px 0;
+  color: #2c2c2c;
+}
+
+article.post h2,
+article.post h3,
+article.post h4 {
+  font-family: 'Inter Tight', sans-serif;
+  font-style: normal;
+  text-align: left;
+  font-weight: 600;
+  font-size: 30px;
+  line-height: 40px;
+  letter-spacing: 0.01em;
+  color: #2c2c2c;
+  margin-bottom: 36px;
+  margin-top: 26px;
+  padding-bottom: unset;
+}
+
+article.post h3 {
+  font-size: 24px;
+}
+
+article.post h4 {
+  font-size: 20px;
+}
+
+article.post blockquote {
+  box-shadow: inset 3px 0px 0px 0px #e0dede;
+  background-color: #f7f7f7;
+  padding: 16px 26px;
+  margin: 40px 0;
+  border-radius: 8px;
+  display: flow-root;
+}
+
+article.post blockquote p {
```

Comment:
can we make the fontsize like 300px or something, and have the marging be almost overlapping the page edge? like a magazine layout. Then the text can be lighter gray, maybe 0.7 line-height italic. and it will not interrupt the reading, instead it serves as sort of a scroll marker